react/components/SearchResult.js

```diff
@@ -153,55 +159,64 @@ export default class SearchResult extends Component {
           </div>
           {!hideFacets && (
             <div className="vtex-search-result__filters">
-              <ExtensionPoint
-                id="filter-navigator"
-                brands={brands}
-                getLinkProps={getLinkProps}
-                map={map}
-                params={params}
-                priceRange={priceRange}
-                priceRanges={priceRanges}
-                query={query}
-                rest={rest}
-                specificationFilters={specificationFilters}
-                tree={tree}
-                hiddenFacets={hiddenFacets}
-                loading={loading && !fetchMoreLoading}
-              />
+              <div className={filterClasses}>
+                <ExtensionPoint
+                  id="filter-navigator"
+                  brands={brands}
+                  getLinkProps={getLinkProps}
+                  map={map}
+                  params={params}
+                  priceRange={priceRange}
+                  priceRanges={priceRanges}
+                  query={query}
+                  rest={rest}
+                  specificationFilters={specificationFilters}
+                  tree={tree}
+                  hiddenFacets={hiddenFacets}
+                  loading={loading && !fetchMoreLoading}
+                />
+              </div>
             </div>
           )}
-          <div className="vtex-search-result__border bg-muted-4 h-75 self-center" />
+          {mobile && <div className="vtex-search-result__border bg-muted-5 h-50 self-center" />} 
           <div className="vtex-search-result__order-by">
-            <OrderBy
-              orderBy={orderBy}
-              getLinkProps={getLinkProps}
-            />
+            <div >
+              <OrderBy
+                orderBy={orderBy}
+                getLinkProps={getLinkProps}
+              />
+            </div>
           </div>
-          <div className="vtex-search-result__gallery">
-            <div className="dn-ns db-s bt b--muted-4">
+          {mobile && <div className="vtex-search-result__border-2 bg-muted-5 h-50 self-center" />}
+          {mobile && <div className="vtex-search-result__switch">
+            <div className="dn-ns db-s">
               <LayoutModeSwitcher
                 activeMode={this.state.galleryLayoutMode}
```

Comment:
Use the variable you destructured in line 113

Reply:
done